docs/HowToDevelopLoadableApps.md

```diff
@@ -0,0 +1,227 @@
+# How To Support Loadable Applications on TizenRT
+
+## CONTENTS
+
+- [Overview of Loadable Apps](#overview-of-loadable-apps)
+- [Assumptions & Limitations](#assumptions-&-limitations)
+- [Enable the existing loadable apps](#enable-the-existing-loadable-apps)
+- [Add new loadable apps configuration](#Add-new-loadable-apps-configuration)
+
+## Overview of Loadable Apps
+- When protected build (CONFIG_BUILD_PROTECTED) is selected, then a separate binary is generated for each user space application apart from a separate kernel binary.
```

Comment:
This is not correct because CONFIG_BUILD_PROTECTED will not generate separate app binaries for each app. 
Instead, let us remove this point and add the following points about loadable apps:
- What is it? (separate binary, stored in flash, loaded to ran during rumtime)
- Why its required? (fault handling, reloading)
- How is it done? (binary loader, elf format)
- Role of binary manager

Reply:
ok I will change this description